### ISSUE TITLE
Add Conversion of all distance units

### ISSUE THREAD

**Issue by Segelzwerg** [Segelzwerg/SportToolBox#87]

**Comment by EEM86** [comment]
@Segelzwerg could you clarify a bit this task? Should I create a new UtilClass for conversion or create a method in Distance interface and implement it?

**Comment by Segelzwerg** [comment]
Yes I was thinking of methods in the interface. I did previously one for Kilometer as reference.
